python/ray/train/lightning/lightning_trainer.py

```diff
@@ -519,9 +527,19 @@ def _lightning_train_loop_per_worker(config):
         logger.warning(
             "`strategy` specified in `LightningConfig.trainer_init_config` "
             "will be ignored. LightningTrainer will create a RayDDPStrategy "
-            "object based on `LightningConfig.ddp_strategy_config`."
+            "or RayFSDPStrategy object based on `LightningConfigBuilder.strategy()`."
```

Comment:
Maybe can remove the implementation details of `RayStrategy`. Just say it will "create a strategy based on the settings passed into LightningConfigBuilder.strategy().`

Reply:
Done